### PR TITLE
Update English introduction - first page

### PR DESCRIPTION
**Pull request by rayzhou2017**

### CODE REVIEW

File: content/v2.1/en/introduction/what-is-kubesphere.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ description: 'What is KubeSphere'
...
+KubeSphere teams not only supports upstream Kubernetes, but also works with its users to alleviate the pain points of Kubernetes including **storage, network, security and ease of use**, and develops new features and capabilities that are not yet available in upstream Kubernetes. KubeSphere provides a complete user experience around Kubernetes that incorporates a set of cloud native ecosystem tools, allowing developers and DevOps teams use their favorite tools in a user-friendly console. KubeSphere delivers **consolidated views while integrating a wide breadth of ecosystem tools** around Kubernetes and offers consistent user experience to reduce complexity. Most importantly, these functionalities are loosely coupled with the platform since they are pluggable and optional at your will. Last but not least, it does not change Kubernetes itself anything. In another word, KubeSphere can be deployed on any existing version-compatible Kubernetes cluster.
```
Comment (FeynmanZhou) [inline content/v2.1/en/introduction/what-is-kubesphere.md]: `use their favorite tools in a user-friendly console.`  => `use their favorite tools in a unified console.` 
I think `unified` can prominent its advantage in this context.

File: content/v2.1/en/introduction/what-is-kubesphere.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ description: 'What is KubeSphere'
...
 
-With KubeSphere container platform, developers can focus on application development since KubeSphere helps your enterprise migrate, deploy and manage existing and containerized apps seamlessly accross **VM, bare metal, Kubernetes, on-premise and public cloud** infrastructures. This platform also helps your Ops team with enterprise-level **observability and troubleshooting, unified monitoring and logging, centralized storage and networking management, easy-to-use CI/CD pipelines**, letting IT operation team embrace DevOps and automated workflows.
+With KubeSphere container platform, developers can focus on application development since KubeSphere helps your enterprise migrate, deploy and manage existing and containerized apps seamlessly across **VM, bare metal, Kubernetes, on-premise and public cloud** infrastructures. This platform also helps your Ops team with enterprise-level **observability and troubleshooting, unified monitoring and logging, centralized storage and networking management, easy-to-use CI/CD pipelines**, letting IT operation team embrace DevOps and automated workflows.
```
Comment (FeynmanZhou) [inline content/v2.1/en/introduction/what-is-kubesphere.md]: VM => virtual machine

 Please do not use acronyms

File: content/v2.1/en/introduction/what-is-kubesphere.md (hunk 1 above)
Comment (FeynmanZhou) [inline content/v2.1/en/introduction/what-is-kubesphere.md]: `KubeSphere teams not only supports upstream Kubernetes` => `KubeSphere not only supports provisioning upstream Kubernetes cluster`
 
KubeSphere Installer can help users to setup a Kubernetes cluster, this is a feature for new k8s users.

Comment (rayzhou2017) [reply content/v2.1/en/introduction/what-is-kubesphere.md]: done

File: content/v2.1/en/introduction/features.md [hunk 3]
```diff
@@ -18,122 +16,122 @@ The following modules elaborate on the key features and benefits provided by Kub
...
-- In the cluster resource dimension, we provide multiple indicators such as CPU utilization, memory utilization, CPU load average, disk usage, inode utilization, disk throughput, IOPS, network card rate, container group running state, ETCD monitoring, API Server monitoring.
-- In the dimension of application resources, we provide five monitoring indicators such as CPU consumption, memory consumption, number of container clusters, network outflow rate and network inflow rate for the application. Besides, we support the query by the amount of sorting and custom time range as well as fast location of exceptions.
-- Provide resource usage ranking by node, enterprise space, project, etc.
-- Provide service component monitoring to quickly locate component failures.
+- Monitoring system is fully visualized, and provides open standard APIs for enterprises to integrate their existing operating platforms such as alerting, monitoring, logging etc. in order to have a unified system for their daily operating work.
+- Comprehensive and second precision monitoring metrics: multi-dimensional metrics with second-level precision and more than 16 metrics.
```
Comment (FeynmanZhou) [inline content/v2.1/en/introduction/features.md]: `Comprehensive and second precision monitoring metrics`

I think `second` should be removed.

Comment (rayzhou2017) [reply content/v2.1/en/introduction/features.md]: second is the time, like hour, minute, second.

**Pull request merged**